docs/troubleshooting-guide/index.md

```diff
@@ -41,6 +41,7 @@ Problems, their causes, how to detect them, and how to fix them.
 - [The dates and times in my questions and charts are wrong][incorrect-times].
 - [My dashboard filters don't work][filters].
 - [My dashboard's linked filters don't work][linked-filters].
+- [My model doesn't work][models].
```

Comment:
Should this section be renamed "Questions, dashboards, and models"? Of should we start a new section called "Models" (I'm also anticipating "Models and metrics").

Reply:
And very soon... "Models, metrics, and custom expressions"? 😆